### DISCUSSION TITLE
Invalid workflows file in github actions

### DISCUSSION THREAD

**Discussion by FLAMESpl** [community/community#23056] [New to GitHub]
<p>Hi, my github action is failing due to error in workflows file: <a href="https://github.com/FLAMESpl/oaklab-blazor-navigation/actions/runs/2707410815/workflow" class="inline-onebox" rel="noopener nofollow ugc">Update README.md · FLAMESpl/oaklab-blazor-navigation@94b8336 · GitHub</a></p>
<p>I cannot tell what is wrong, can anyone help me?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You’re missing a colon after <code>workflow_call</code>. <code>on</code> is a mapping, so <code>workflow_call</code> must be a key in the mapping even if its value is empty. What you currently have there is a mapping (with keys <code>push</code> and <code>pull_request</code>) mixed with a plain string <code>workflow_call</code>, which makes it invalid YAML.</p>
<p>See also: <a href="https://yaml.org/spec/1.2.2/#21-collections" class="inline-onebox">YAML Ain’t Markup Language (YAML™) revision 1.2.2</a></p>
